suite-native/module-authenticity-checks/src/DeviceCompromisedBannerAtoms.ts

```diff
@@ -0,0 +1,22 @@
+import { atom } from 'jotai';
+
+/**
+ * Represents the display variants for the DeviceCompromisedBanner.
+ * - 'none': Banner is hidden
+ * - 'brief': Shows banner with only title
+ * - 'extended': Shows banner with title, subtitle and CTA
+ */
+export type DeviceCompromisedBannerVariant = 'none' | 'brief' | 'extended';
+
+export type DeviceCompromisedBanner = {
+    variant: DeviceCompromisedBannerVariant;
+};
+
+export const deviceCompromisedBannerAtom = atom<DeviceCompromisedBanner>({ variant: 'none' });
```

Comment:
Do you expect that some other attributes will be added later? If not I believe that the type of the Atom could be a simple string reflecting the variant itself.
```suggestion
export const deviceCompromisedBannerAtom = atom<DeviceCompromisedBanner>('none');
```

Reply:
Ok. I though it good practice to make it extensible - I considered the new atom analogical to a new React Context, where I'd be inclined to do the same, but it's probably not needed :slightly_smiling_face:

Reply:
Done in 22deea04